<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionScreen</class>
 <widget class="QWidget" name="ConnectionScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #1E1E1E;
    color: white;
}





</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="lblConnection">
     <property name="styleSheet">
      <string notr="true">
QLabel {
    color: #00FF00; 
    font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Connection Status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnClear">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #5E81AC; /* Mavi Ton */
    color: white;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #81A1C1; /* Açık Mavi */
}

QPushButton:pressed {
    background-color: #4C566A; /* Koyu Gri */
}
</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextEdit" name="txtLog">
     <property name="styleSheet">
      <string notr="true">
QTextEdit {
    background-color: black;
    color: lightgreen;
    border: 1px solid gray;
    padding: 5px;
}</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btnScan">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #5E81AC; /* Mavi Ton */
    color: white;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}


QPushButton:pressed {
    background-color: #4C566A; /* Koyu Gri */
}
</string>
     </property>
     <property name="text">
      <string>Scan Devices</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnDisconnect">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #5E81AC; /* Mavi Ton */
    color: white;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #81A1C1; /* Açık Mavi */
}

QPushButton:pressed {
    background-color: #4C566A; /* Koyu Gri */
}
</string>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnConnect">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #5E81AC; /* Mavi Ton */
    color: white;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #81A1C1; /* Açık Mavi */
}

QPushButton:pressed {
    background-color: #4C566A; /* Koyu Gri */
}
</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>200</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhNone</set>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxVisibleItems">
      <number>5</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertPolicy::InsertAfterCurrent</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
